flake-modules/dev/init-script/src/main.rs

```diff
@@ -0,0 +1,3 @@
+fn main() {
+    println!("Hello, world!");
```

Comment:
oooh we doin rust now :)

Reply:
I'm a huge noob, but I just want to do a simple `nix-init` clone for nixvim.

Reply:
Not opposed to learning a bit of rust for this, just conscious I already have to improve my python to properly be capable of maintaining our other in-tree scripts... :sweat_smile: